server/routes/bookedtoday/summaryWithRecordController.ts

```diff
@@ -7,12 +7,8 @@ export default class SummaryWithRecordController {
   public view(): RequestHandler {
     return async (req, res) => {
       const { id } = req.params
-      const arrival = await this.expectedArrivalsService.getArrival(id)
-      const singleMatch = arrival.potentialMatches[0]
-      const prisonerSummaryDetails = await this.expectedArrivalsService.getPrisonerSummaryDetails(
-        singleMatch.prisonNumber
-      )
-      return res.render('pages/bookedtoday/summaryWithRecord.njk', { arrival, prisonerSummaryDetails })
+      const data = await this.expectedArrivalsService.getArrivalAndSummaryDetails(id)
```

Comment:
refactored to call single service method in controller and return arrival and summary as one object.